client/components/data/query-site-monitor-settings/README.md

```diff
@@ -0,0 +1,37 @@
+Query Site Monitor Settings
+================
+
+`<QuerySiteMonitorSettings />` is a React component used in managing network requests for retrieving Monitor settings for a Jetpack site.
+
+## Usage
+
+Render the component, passing `siteId`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```jsx
+import React from 'react';
+import QuerySiteMonitorSettings from 'components/data/query-site-monitor-settings';
+
+export default function ExampleSiteComponent( { siteMonitorSettings, translate } ) {
+	return (
+		<div>
+			<QuerySiteMonitorSettings siteId={ 12345678 } />
+			{
+				siteMonitorSettings && siteMonitorSettings.monitor_active
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/13169/commits/4cae3f8c5945f5e86132ff0791cefa47cc7dc7f5.